### DISCUSSION TITLE
Disable node selection

### DISCUSSION THREAD

**Discussion by zldavis** [xyflow/xyflow#4214] [Q&A] [answered]
It's possible I'm missing this from the documentation, but as far as I can tell there is not a way to disable selection in general. Would be useful to have a prop like `nodesSelectable: boolean` which can be used to disable any selections.

**Comment by bcakmakoglu** [comment, marked as answer]
There is no prop for nodes specifically but there is a prop that can disable selection of *elements* (nodes and edges) globally: [`elementsSelectable`](https://reactflow.dev/api-reference/react-flow#elements-selectable).

```tsx
<ReactFlow elementsSelectable={false} ...>
```

**Reply by zldavis** [reply]
🤔 hmm not sure how I missed that, thanks!
